docs/5.0/preview/cloud.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Teleport Cloud Preview
+description: Teleport hosted and managed by Teleport team.
+---
+
+# Teleport Cloud Preview
+
+Teleport cloud is a managed an hosted service of Teleport.
+Request access to preview [here](https://goteleport.com/get-started/).
+
+## Status
+
+Teleport cloud is currently only recommended for environments that do not require
+meeting strict compliance standards or SLA uptime guarantees.
+
+We believe that Teleport cloud environment is secure, however we scrutinizing
```

Comment:
`however we scrutinizing` reads a bit weird to me. Maybe something more like `however, we are in the process of scrutinizing and executing...`